media/css/test.scss

```diff
@@ -0,0 +1,4 @@
+#nonexistent ul > li {
+  display:inline-block;
+}
+#nonexistent span { color:#fff;}
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Avoid using id selectors<br>Colon after property should be followed by one space